src/views/Pools/components/BountyCard.tsx

```diff
@@ -26,25 +27,18 @@ const StyledCard = styled(Card)`
   }
 `
 
-const InlineText = styled(Text)`
+const InlineBalance = styled(Balance)`
   display: inline;
```

Comment:
I've created a PR to allow the `Text` component to receive an `inline` prop. Reasons I think this would be useful are within the PR. It would also mean I can pass the same to the `<Balance>` component, and not need to restyle like this

https://github.com/pancakeswap/pancake-toolkit/pull/96

Reply:
It got merged and UIKit bumped. 👍

Reply:
Amazing - this has neatened stuff up a lot 🙌